hotel_daily_service/wizard/hotel_daily_service_wizard.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from dateutil import parser
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class HotelDailyServiceLineWizard(models.TransientModel):
+    _name = "hotel.daily.service.line.wizard"
+    _description = "Hotel Daily Service Line"
+
+    @api.model
+    def _default_hotel_service_id(self):
+        breakfast = self.env.ref("hotel_daily_service.hotel_service_breakfast")
+        return breakfast
+
+    hotel_folio_id = fields.Many2one(
+        comodel_name="hotel.folio", string="Folio"
+    )
+    hotel_service_id = fields.Many2one(
+        comodel_name="hotel.services",
+        string="Service",
+        default=_default_hotel_service_id,
```

Comment:
`default=_default_hotel_service_id` Instead of the call the method, you can use as 
default=lambda self: self.env.ref('hotel_daily_service.hotel_service_breakfast'))

Reply:
That's cleaner, indeed. Thanks!